tests/filecheck/dialects/transform/transform_interpreter.mlir

```diff
@@ -0,0 +1,17 @@
+// RUN: xdsl-opt %s -p transform-interpreter | filecheck %s
+
+
+module {
+
+  "builtin.module"() ({
+    "transform.named_sequence"() <{arg_attrs = [{transform.readonly}], function_type = (!transform.any_op) -> (), sym_name = "__transform_main"}> ({
```

Comment:
why not custom syntax here?

Reply:
I'd rather the tests were using custom syntax if possible, adding it in a separate PR would be nice to make the tests more readable.

Reply:
The named sequence op [does not yet have a custom syntax.](https://github.com/xdslproject/xdsl/blob/main/xdsl/dialects/transform.py#L737-L782) I can change the other ones, but I rather have small commits than a big one that does many changes, so the custom syntax can be a different commit.